From dbaaf2b294af120e42a0e95644640b128044a568 Mon Sep 17 00:00:00 2001
From: Simon Wunderlich <sw@simonwunderlich.de>
Date: Wed, 23 Oct 2013 01:17:36 +0200
Subject: [PATCH] alfred-gpsd: add -lm to linker flags for libgps

Appearently applications using libgps must link -lm themselves. A patch
was submitted [1] to change this unusual behaviour upstream, but was
never applied. Therefore link to -lm to alfred-gpsd too as this should
not hurt anyway.

[1] https://lists.berlios.de/pipermail/gpsd-dev/2011-August/009451.html

Signed-off-by: Simon Wunderlich <sw@simonwunderlich.de>
---
 gpsd/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gpsd/Makefile b/gpsd/Makefile
index 9b21652..92e690f 100644
--- a/gpsd/Makefile
+++ b/gpsd/Makefile
@@ -49,7 +49,7 @@ ifeq ($(origin LIBGPS_CFLAGS) $(origin LIBGPS_LDLIBS), undefined undefined)
     $(error No $(LIBGPS_NAME) development libraries found!)
   endif
   LIBGPS_CFLAGS += $(shell $(PKG_CONFIG) --cflags $(LIBGPS_NAME))
-  LIBGPS_LDLIBS +=  $(shell $(PKG_CONFIG) --libs $(LIBGPS_NAME))
+  LIBGPS_LDLIBS +=  $(shell $(PKG_CONFIG) --libs $(LIBGPS_NAME)) -lm
 endif
 CFLAGS += $(LIBGPS_CFLAGS)
 LDLIBS += $(LIBGPS_LDLIBS)
-- 
1.7.10.4

